psapi.go

```diff
@@ -98,3 +99,33 @@ func EnumProcesses() (pids []uint32, err error) {
 		}
 	}
 }
+
+// PerformanceInformation represents the PERFORMANCE_INFORMATION structure
+type PerformanceInformation struct {
+	CB                uint32
```

Comment:
Is there any reason to export `CB`? I noticed it wasn't exported in https://github.com/elastic/go-windows/blob/ee94ac30503b3ecb77e35710f8648d500919f3b9/psapi.go#L43

Reply:
it looks like it is exported in that one?

Reply:
First letter is lowercase so that is unexported.

https://go.dev/ref/spec#Exported_identifiers